python/lsst/ip/diffim/psfMatch.py

```diff
@@ -366,6 +366,17 @@ class PsfMatchConfig(pexConfig.Config):
         doc="""Use Bayesian Information Criterion to select the number of bases going into the kernel""",
         default=False,
     )
+    fwhmExposureGrid = pexConfig.Field(
+        doc="Grid size to compute the average FWHM in an exposure",
+        dtype=int,
+        optional=True,
+    )
+    fwhmExposureBuffer = pexConfig.Field(
+        doc="Fractional buffer margin to be left out of all sides of the image during construction"
+            "of grid to compute average FWHM in an exposure",
+        dtype=float,
+        optional=True
+    )
```

Comment:
I added these config options in the base class here. But I now realize these should probably be just in the task that subtract images.